workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -222,9 +220,7 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
             continue@outer
           }
 
-          // Make sure the runtime has not been cancelled from runner.processAction()
-          if (!isActive) return@launch
-
+          // Render pass for the updated state from the action applied.
```

Comment:
Seems like you're confident that `applyNextAvailabletreeAction()` cannot make us inactive, how come?